### PR TITLE
cli: add new rollup plugin for forwarding file imports

### PR DESCRIPTION
**Pull request by Rugvip**
Our published packages aren't importing image assets properly, because the current plugin translates the import into something that doesn't make sense for webpack.

I searched around quite a bit, but wasn't able to find any Rollup plugin that can take a simple

```ts
import MicDrop from './mic-drop.svg'
```

And then copy the file and keep it as something similar to this in the output

```ts
import MicDrop from './layout/ErrorPage/mic-drop.svg'
```

But couldn't find anything, so made our own instead

It's possible this could be useful for other projects, especially if it doesn't exist as opposed to me not being able to find it. In that case we could extract it out of the project.

Would also love to hear that this is simple to solve through config and we can delete all this... please? 😅

### CODE REVIEW

**Comment by stefanalund** [comment]
Where can I watch Futurama?

**Comment by Fox32** [comment]
In Germany you get it with Amazon Prime 😉

File: packages/cli/src/lib/builder/plugins.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  // We keep track of output directories that we've already copied files
+  // into, so that we don't duplicate that work
+  const generatedFor = new Set<string>();
+
+  return {
+    name: 'forward-file-imports',
+    async generateBundle(outputOptions, bundle, isWrite) {
+      if (!isWrite) {
+        return;
+      }
```
Comment (Fox32) [inline packages/cli/src/lib/builder/plugins.ts:56]: Codecov is just a little bit annoying 😆

**Comment by freben** [comment]
Well of course windows fails

File: packages/cli/src/lib/builder/plugins.ts (hunk 1 above)
Comment (Rugvip) [reply packages/cli/src/lib/builder/plugins.ts:56]: Well now it's covered :>

**Review by freben — APPROVED** [review]

**Pull request merged**